### PR TITLE
Documentation/40139 adding new faq

### PR DESCRIPTION
**Pull request by birthe**

### CODE REVIEW

File: docs/faq/README.md [hunk 1]
```diff
@@ -190,6 +196,10 @@ No, it's not possible to use OpenProject without Internet access (Enterprise clo
 
 Yes, that’s possible. Please have a look at our [Excel sync](../user-guide/integrations/excel-synchronization).
 
+### Is possible to create a task in one place and have it show up in several projects (and only have to update from one place). If so, how can I go about configuring that?
+
+As a workaround you could use work package relations. Once the task is created in one project you can create the relation "includes" in another project. This would be an option to display that task in multiple projects. 
+
```
Comment (birthe) [inline docs/faq/README.md:263]: Link to the section for Work package relations.

**Review by birthe — COMMENTED** [review]
HI @JanKeppler looks good to me. See one comment .

File: docs/faq/README.md [hunk 2]
```diff
@@ -138,10 +171,22 @@ If you are using Enterprise on-premises, please write an email to sales @ openpr
...
 You can do this as a system administrator in the [System settings](../system-admin-guide/system-settings/display-settings/#time-and-date-formatting-aggregation-of-changes-in-activity).
 
+### How to use plugins in OpenProject and where can I find them?
+
+Not all available plugins for OpenProject are published yet. After updating the plugins so that they are working with the Rails3 OpenProject Core, they will be published on Github continiously. Please write us a note if you have questions to any of the plugins.
+
+### Which Plugins are published so far?
+
+Please refer to the project timeline to see which plugins will be released. Plugin development will take place in dedicated sub-projects on openproject.org.
+
```
Comment (birthe) [inline docs/faq/README.md:193]: Please refer to the list of [integrations and Community plugins](../system-admin-guide/integrations/).

**Comment by oliverguenther** [comment]
This PR is a duplicate of https://github.com/opf/openproject/pull/9900

**Pull request closed without merging**